ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1017,6 +1017,30 @@ def cdread(self, *args, **kwargs):
 
         self.input(fname, **kwargs)
 
+    @wraps(_MapdlCore.tbft)
+    def tbft(self, oper='', id_='', option1='', option2='', option3='', option4='', option5='', option6='', option7='', **kwargs):
+
```

Comment:
Always include a docstring, even if wrapping.

```suggestion
        """Wraps ``_MapdlCore.tbft``."""
```